### ISSUE TITLE
Invalid error handling

### ISSUE THREAD

**Issue by toshiyukihina** [inventit/moatjs-stub#2]
Invalid error handling is here: 
https://github.com/inventit/moatjs-stub/blob/master/moat.js#L228

```
        var descriptor = packageJson.models[type];
        if (!type) {
          throwError('Unknown type:' + type);
        }
```

Fixed to be as follows:

```
        var descriptor = packageJson.models[type];
        if (!descriptor) {
          throwError('Unknown type:' + type);
        }
```

**Comment by toshiyukihina** [comment]
Fixed in aa338770551bb24453a6d94e1b444123c75f65cc.
Not yet released.

**Comment by toshiyukihina** [comment]
Released as v1.2.2
